app/src/main/res/values/strings.xml

```diff
@@ -147,29 +147,29 @@
     <string name="moveBarcodeToCenterOfScreen">Center the barcode on the screen</string>
 
     <string name="noBarcodeFound">No barcode was found</string>
-    <string name="errorReadingImage">Error reading image</string>
+    <string name="errorReadingImage">Could not read the image</string>
 
     <string name="balance">Balance</string>
     <string name="currency">Currency</string>
     <string name="points">Points</string>
 
     <string name="parsingBalanceFailed"><xliff:g>%s</xliff:g> does not seem to be a valid balance.</string>
-    <string name="chooseImportType">Import data from which app?</string>
+    <string name="chooseImportType">Import data from?</string>
     <string name="app_loyalty_card_keychain">Loyalty Card Keychain</string>
 
     <string name="privacy_policy">Privacy Policy</string>
-    <string name="privacy_policy_popup_text">Many app stores require apps to show their privacy policy on first start. Here is ours:\n\nWe collect NO DATA AT ALL and our app is Open Source so anyone can confirm this to be true.</string>
+    <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nWe collect NO DATA AT ALL, which anyone can confirm since our app is libre software.</string>
     <string name="accept">Accept</string>
     <string name="importCatima">Import from Catima</string>
     <string name="importCatimaMessage">Please select your Catima export file. It is most likely named Catima.csv.\n\nA Catima export file can be created by going to the Import/Export menu and pressing "Export".</string>
-    <string name="importFidme">Import from Fidme</string>
-    <string name="importFidmeMessage">Please select your Fidme export file. It is most likely named something like fidme-export-request-xxxxxx.zip.\n\nA Fidme export file can be created in the Fidme app by going to your profile, choosing "Data Protection" and then pressing "Extract my data".\n\nPlease note that Fidme does not store the barcode type in the export data so you will have to edit the imported cards manually.</string>
+    <string name="importFidme">Import from FidMe</string>
+    <string name="importFidmeMessage">Please select your FidMe export file. It is most likely named something like fidme-export-request-xxxxxx.zip.\n\nA Fidme export file can be created in the Fidme app by going to your profile, choosing "Data Protection" and then pressing "Extract my data".\n\nPlease note that Fidme does not store the barcode type in the export data so you will have to edit the imported cards manually.</string>
```

Comment:
The funny part is that FidMe themselves aren't even consistent.

English Google Play: FidMe
French Google Play: FIDME
Their website: Fidme
Installed app in Android launcher: fidme

But it does make sense to use the English Google Play name. However, if we do, let's be consistent and change it everywhere in the string (you left several Fidme references)

Reply:
Fixed, my mistake. "FidMe-export-request-xxxxxx.zip" remains somewhat dubious.

Reply:
fidme-export-request-xxxxxx.zip should definitely be lowercase because the filename their server gives you is named as such.